Mass patching script (#423)
Gizra/drupal-starter: README.md

Follow the steps listed in `.ddev/providers/pantheon.yaml`.
Make sure to add the correct site name under `environment_variables.project`.

> There's an experimental Robo command to do the full process of creating a new project:

Reviewer: ```suggestion
There's a Robo command to do the entire process of creating a new project:
```

```
ddev robo bootstrap:project newname git@github.com/newclient/newproject.git terminus_token github_token
```

> See https://github.com/Gizra/drupal-starter/blob/main/robo-components/BootstrapTrait.php
> for details.

Reviewer: ```suggestion
See the details [here](https://github.com/Gizra/drupal-starter/blob/main/robo-components/BootstrapTrait.php).
```

> Also as this repository gets copied several times, it gets tedious to port even small fixes.
> For larger-scale changes, due to conflicts and per-project considerations, we need to apply
> changes manually to the cloned repositories, but for tiny, trivial changes, we have the following tool:

Reviewer: ```suggestion
As this repository gets copied several times, for different projects, it gets tedious to port small fixes.
For larger-scale changes, due to conflicts and per-project considerations, we need to apply
changes manually., However for tiny, trivial changes, such as Travis fixes, we have the following tool:
```

```

> cd /home/user/clientprojects

Reviewer: ```suggestion
# Go to the root of all the projects
cd /home/user/your-projects
```

> export REPOSITORIES=[client1 client2 client3]

Reviewer: ```suggestion
# If no export is provided, command will ask entering manually the names.
export REPOSITORIES=[client1 client2 client3]
```

> /path/to/starter/scripts/mass_patch.sh [gh_token_that_can_create_prs] /tmp/our-little-patch

Reviewer: Maybe change to something that would allow us to have a better PR title

```suggestion
./drupal-starter/scripts/mass_patch.sh [gh_token_that_can_create_prs] "PR title"
```

Author: That could be an additional argument, that's right.

```

> This needs to be executed natively.
> It will try to refresh the working copies there and apply the patch, if it succeeds, it opens
> a pull request in the repository.
> That provides a fast track to spread changes for those parts of the Starter Kit what typically
> remain unchanged after cloning (CI scripts, testing scripts, DDEV configuration and commands and so on).

Reviewer: ```suggestion
This needs to be executed natively (i.e. no inside `ddev ssh`)
It will try to refresh the working copies there and apply the patch. If it succeeds, it opens
a pull request in the destination repository.
That provides a fast track to spread changes for those parts of the Starter Kit that typically
remain unchanged after cloning (CI scripts, testing scripts, DDEV configuration and commands, and so on).
```
